### PR TITLE
update drinks, added comments and Norm

### PR DESCRIPTION
**Pull request by GrooGrux**

### CODE REVIEW

**Comment by GrooGrux** [comment]
Ok mate, it's a better beer.  and i added one for you as well.

File: modules/botExtensions/jokesExtension.js [hunk 1]
```diff
@@ -42,6 +44,22 @@ module.exports = function (txt, msg, postMessage) {
             return;
         }
 
+        //Easter Egg jokes:
+
+        if (/^Norm!$/.test(txt)) {
```
Comment (adylevy) [inline modules/botExtensions/jokesExtension.js:49]: this will not work, text should be lowercase.

File: modules/botExtensions/jokesExtension.js [hunk 2]
```diff
@@ -42,6 +44,22 @@ module.exports = function (txt, msg, postMessage) {
...
+            postMessage('Evening Everybody');
+        }
+
+        if (/^How\sabout\sa\sbeer\snorm$/.test(txt)) {
+            postMessage('OK, but stop me at one...  Make that one thirty.');
+        }
+
+        if (/^Whats\sshakin\snorm$/.test(txt)) {
```
Comment (adylevy) [inline modules/botExtensions/jokesExtension.js:57]: same

File: modules/botExtensions/jokesExtension.js [hunk 3]
```diff
@@ -42,6 +44,22 @@ module.exports = function (txt, msg, postMessage) {
             return;
         }
 
+        //Easter Egg jokes:
+
+        if (/^Norm!$/.test(txt)) {
+            postMessage('Evening Everybody');
+        }
+
+        if (/^How\sabout\sa\sbeer\snorm$/.test(txt)) {
```
Comment (adylevy) [inline modules/botExtensions/jokesExtension.js:53]: same

**Comment by GrooGrux** [comment]
output messages from Raven must be in lower case?

**Pull request closed without merging**